test/jit/test_schema_check.py

```diff
@@ -16,6 +18,50 @@
                        "\tpython test/test_jit.py TESTNAME\n\n"
                        "instead.")
 
+# This TorchDispatchTensor Subclass is used to test incorrectly aliasing ops
+# This is done by aliasing the output to the first argument for all ops in INCORRECT_OPS
```

Comment:
minor nit: the comment says that this subclass is used to test incorrectly aliasing ops, but technically it's `SchemaCheckMode` that's responsible for that. It looks like the main purpose of this class is to test that `SchemaCheckMode` behaves as expected, by "simulating" an op with an incorrect schema and asserting that the mode picks up the error.

Do you mind updating the description a bit?

Can you make it a bit clearer in this comment that the purpose of this class is basically as a test case for `schema_check_mode`? It doesn't look like it's act